payments/invoice/other-features.md

```diff
@@ -281,5 +281,5 @@ next_href="after-payment" next_title="Next: After Payment" %}
 
 [fi-png]: /assets/img/fi.png
 [callback-api]: #callback
-[invoice-flow]: /payments/invoice/index/#invoice-flow
+[invoice-flow]: /payments/invoice/index#invoice-flow
```

Comment:
I don't think we need to explicitly name `index` in the URL, as it will be implicitly used if the path segment is blank.

```suggestion
[invoice-flow]: /payments/invoice#invoice-flow
```

Reply:
Added this to Invoice task in Jira